enhancements/kube-apiserver/stability-point-to-point-network-check.md

```diff
@@ -82,6 +82,80 @@ What are the caveats to the implementation? What are some important details that
 didn't come across above. Go in to as much detail as necessary here. This might
 be a good place to talk about core concepts and how they relate.
 
+#### RelatedEndpoint Custom Resource
+
+```go
+package v1alpha1
+import 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+// RelatedEndpoint references a "interesting" endpoint for a given pod.
+type RelatedEndpoint struct {
```

Comment:
choose a group.  `network.openshift.io` maybe.  Try to work out sharing it with sdn.

`ConnectionCheck` as the resource maybe.